VERSION

```diff
@@ -1 +1 @@
-2.1.26
+2.1.25
```

Comment:
Was this accidental change?

Reply:
I accidentally bumped the version in previous PR's and forgot to return it back :)